src/views/results/html.ts

```diff
@@ -141,6 +141,15 @@ export function generateScroller(basicSelect: string, isCL: boolean): string {
             });
           }
 
+          function isJsonString(str) {
```

Comment:
I think this `isJsonString` should be converted to be more like `getFormattedString`, where we could pass JSON, XML, etc, later on, and have it return the correct format. Then, in the catch, return the original `str`. Also, `str` here is missing the type on the parameter.